pro_tes/middleware/middleware.py

```diff
@@ -27,10 +29,23 @@ class TaskDistributionMiddleware(AbstractMiddleware):
 
     def __init__(self):
         """Construct object instance."""
-        self.tes_uri = task_distribution()
+        self.tes_uri = []
+        self.input_uri = []
 
     def modify_request(self, request) -> Dict:
-        """Add TES instance to request body."""
+        """Add the best possible TES instance to request body."""
```

Comment:
I think it would be best if the user/admin could change the task distribution logic to be used (currently `random` or `distance`). So please add a config param to `config.yaml` that points to the module to be used for task distribution (e.g., `pro_tes.middleware.task_distribution.distance` (compare comment below). This would then also allow using a task distribution logic from an external package.

We will need this setting so that we can compare the efficiency of different task distribution models.